d2l-tspan.html

```diff
@@ -69,18 +104,45 @@
 			attached: function() {
 				var slot = Polymer.dom(this.root).querySelector('#float');
 				this._sizeObserver.observe(slot);
+				slot.addEventListener('focusout', this._boundFocusOutHandler);
```

Comment:
If event handlers are wired up in `attached`, they should be unwired in `detached`.  In addition, when wiring up event handlers in `attached`, it is preferred to wired up in an `afterNextRender` to defer that work to the next frame.  This is generally possible for events that depend on user input, because we would not expect the event handler to fire until at least 25-50 frames later.